src/styles/_buttons.scss

```diff
@@ -33,10 +37,10 @@
   &:not(:disabled):not(.disabled):active:focus {
     box-shadow: none;
   }
-  // TODO: Uncomment this if everyone agrees that disabled btn should be in this style
-  /* &:disabled {
-    background-color: $background-color;
-    color: $border-color;
+
+  &:disabled {
+    background-color: darken($background-color, 7.5%);
+    color: darken($border-color, 7.5%);
```

Comment:
I've made these changes as the designer's colors arent meant to be used on the background, which we use.